Gemfile

```diff
@@ -29,8 +29,6 @@ gem 'bootstrap_form'
 gem 'font-kit-rails'
 gem 'nprogress-rails'
 gem 'font-awesome-rails', '~> 4.7.0'
-gem "paperclip", "~> 5.2"
-gem 'delayed_paperclip'
```

Comment:
Can CarrierWave be configured so that it processes the images in a background job?

Reply:
Tried [carrierwave_backgrounder](https://github.com/lardawge/carrierwave_backgrounder) but it didn't seem to want to work and hasn't been updated in quite a while.

There is [carrierwave_direct](https://github.com/dwilkie/carrierwave_direct) which works through providing direct uploads to S3.